scp/lib/bfs/bucket/scp.rb

```diff
@@ -93,7 +96,7 @@ def open(path, encoding: self.encoding, tempdir: nil, **_opts, &block)
       # Deletes a file.
       def rm(path, **_opts)
         path = full_path(path)
-        sh! 'rm', '-f', path
+        sh! %(rm -f #{Shellwords.escape(path)})
```

Comment:
Question: why, btw? Or is it rubocop? I'd prefer original variant.

Reply:
me too, but the `-exec stat {} \;` addition doesn't work since we were trying to escape that